### PR TITLE
v1.0.1 release 

### PR DESCRIPTION
**Pull request by rnpridgeon**

### CODE REVIEW

File: tools/build-manylinux.sh [hunk 1]
```diff
@@ -44,7 +44,7 @@ fi
 #
 
 echo "# Installing basic system dependencies"
-yum install -y zlib-devel gcc-c++
+yum install -y zlib-devel gcc-c++ƒ
```
Comment (edenhill) [inline tools/build-manylinux.sh]: this looks weird

Comment (rnpridgeon) [reply tools/build-manylinux.sh]: Interesting I thought I pushed the removal of this prior to opening a pr

Comment (edenhill) [reply tools/build-manylinux.sh]: Good thing there is a web page showing what has actually been pushed.
This one
;)

**Comment by rnpridgeon** [comment]
OSX build failure indicated curl exited with error code 56 which indicates we hit a failure receiving data. This seems like it should be intermittent so I reran the task.

**Comment by Feandil** [comment]
Is there any plan to release this in the close future?

**Pull request merged**